jenkins-build.sh

```diff
@@ -64,3 +64,5 @@ if type sloccount >/dev/null 2>&1
 then
 	make sloccount.sc
 fi
+
+bash -ex -c 'cd examples/stamps/; ./do.sh'
```

Comment:
Oh, I just had an idea: if this test doesn't work there won't be a good indication of why

if you add `|| { echo examples/stamps/do.sh no longer runs; /bin/false }` to the end of line 68 that will fix this.